aspnetcore/signalr/hubs.md

```diff
@@ -37,9 +37,29 @@ Create a hub by declaring a class that inherits from `Hub`, and add public metho
 
 You can specify a return type and parameters, including complex types and arrays, as you would in any C# method. SignalR handles the serialization and deserialization of complex objects and arrays in your parameters and return values.
 
+## The Context object
+
+Each instance of the `Hub` class has a property named `Context` that contains the following properties with information about the connection:
+
+| Property | Description |
+| ------ | ----------- |
+| `ConnectionId` | Gets the unique ID for the connection, assigned by SignalR. There is one connection ID for each connection.|
+| `UserIdentifier` | Gets the user identifier. By default, SignalR uses the `ClaimTypes.NameIdentifier` from the `ClaimsPrincipal` associated with the connection as the user identifier. |
+| `User` | Gets the `ClaimsPrincipal`. |
+| `Items` | Gets a key/value collection that can be used to share data within the scope of this connection. You can store items in this collection and they will persist for that connection across different hub method invocations. |
+| `Features` | Gets the collection of [HTTP features](xref:fundamentals/request-features) available on the connection. |
```

Comment:
Actually this isn't quite correct. It doesn't get the collection of HTTP features, it gets a collection of "connection features" available as part of the SignalR connection. SignalR uses the same basic abstraction but the content of this collection is *different* from `HttpContext.Features`. This is a very advanced feature that I wouldn't expect customers to need to use though.

Reply:
I don't have a great idea on how to document this without going in to a whole new level of detail :). Honestly I just want to tell users "don't worry about this one for now" ;P. At some point, as we expand SignalR and move in to non-HTTP spaces, this might become more relevant but for now there's very little a user can do here that they can't already do elsewhere.